packages/myst-to-react/src/MyST.tsx

```diff
@@ -28,11 +28,12 @@ export function MyST({
   className?: string;
 }) {
   const renderers = useNodeRenderers();
-  if (!ast || ast.length === 0) return null;
+  if (!ast) return null;
   if (!Array.isArray(ast)) {
     const Component = selectRenderer(renderers, ast);
     return <Component key={ast.key} node={ast} className={className} />;
   }
+  if (ast.length === 0) return null;
```

Comment:
Minor updates to this logic, technically a node with `{length: 0}` as a property would never render. This ensures first that we are dealing with an array.